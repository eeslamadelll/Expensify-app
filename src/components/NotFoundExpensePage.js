import React from 'react';
import {Link} from 'react-router-dom';

const NotFoundExpensePage = () => {
    return (
        <div>
            404 Not Found Page! <Link to="/">Back Home</Link>
        </div>
    );
}

export default NotFoundExpensePage;