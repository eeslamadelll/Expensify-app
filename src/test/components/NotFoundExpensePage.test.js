import React from 'react';
import { shallow } from 'enzyme';
import NotFoundExpensePage from '../../components/NotFoundExpensePage';

test('should render NotFoundExpensePage', () => {
	const wrapper = shallow(<NotFoundExpensePage />);
	expect(wrapper).toMatchSnapshot();
});