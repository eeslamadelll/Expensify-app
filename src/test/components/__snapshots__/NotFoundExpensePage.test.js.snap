// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render NotFoundExpensePage 1`] = `
<div>
  404 Not Found Page! 
  <Link
    replace={false}
    to="/"
  >
    Back Home
  </Link>
</div>
`;
